src/firestore/firekit.ts

```diff
@@ -1153,6 +1154,7 @@ export class RoarFirekit {
     isTestData = false,
   }: {
     name: string;
+    publicName: string;
```

Comment:
Let's make it optional
```suggestion
    publicName?: string;
```